pkg/packetfilter/iptables/iptables.go

```diff
@@ -327,9 +307,7 @@ func ToRuleSpec(rule *packetfilter.Rule) ([]string, error) {
 		ruleSpec = append(ruleSpec, "--set-mark", rule.MarkValue)
 	}
 
-	logger.Infof("ToRuleSpec: %s", strings.Join(ruleSpec, " "))
```

Comment:
I think it would be useful to print ruleSpec here for debugging

Reply:
OK - we can leave it in as TRACE - I think as just Info level it might fill logs too much.

Reply:
Let's keep it as TRACE , to make sure we didn't miss something in packetfilter rule to iptables rule translation

Reply:
I pushed commit https://github.com/submariner-io/submariner/pull/2925/commits/a0c44f52a5d9021e1450ad9922269ec7a4a533c6.  I put the TRACE log back in and also added the rule spec string to the error logging.

Reply:
Also added a `String` method for `Rule`: https://github.com/submariner-io/submariner/pull/2925/commits/21e672b4c73a32d67d9ce62acca782a2ea91fec2.